tests/unit_tests/test_wave_06.py

```diff
@@ -32,13 +32,20 @@ def test_get_no_matching_items_by_category():
     )
 
     items = vendor.get_by_category("Electronics")
+    # assert
 
-    raise Exception("Complete this test according to comments below.")
+    assert item_a.get_category() != "Electronics"
+    assert item_b.get_category() != "Electronics"
+    assert item_c.get_category() != "Electronics"
+    assert not items
+    assert len(items) == 0 
```

Comment:
Only the second assert is needed. It is more accurate to what we are looking for and if len(items) is 0, we know for sure that it's falsey, so we don't need the first assert.